packages/starlight/components/SidebarSublist.astro

```diff
@@ -65,6 +65,10 @@ interface Props {
 		color: var(--sl-color-white);
 	}
 
+	.top-level > li {
+		overflow-wrap: anywhere;
+	}
```

Comment:
Should this really only apply to the top-level list, or any `li`?

Reply:
The components being black boxes, it's safe to apply the rule directly to `li` as defensive CSS. I've changed it to:

```diff
diff --git a/packages/starlight/components/SidebarSublist.astro b/packages/starlight/components/SidebarSublist.astro
index c4c5882..63308f6 100644
--- a/packages/starlight/components/SidebarSublist.astro
+++ b/packages/starlight/components/SidebarSublist.astro
@@ -53,6 +53,10 @@ interface Props {
                padding: 0;
        }
 
+       li {
+               overflow-wrap: anywhere;
+       }
+
        ul ul li {
                margin-inline-start: var(--sl-sidebar-item-padding-inline);
                border-inline-start: 1px solid var(--sl-color-hairline-light);
@@ -65,10 +69,6 @@ interface Props {
                color: var(--sl-color-white);
        }
 
-       .top-level > li {
-               overflow-wrap: anywhere;
-       }
-
        .top-level > li + li {
                margin-top: 0.75rem;
        }
```